examples/notebook/notebook.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: substratus.ai/v1
+kind: Notebook
+metadata:
+  name: example
+spec:
+  image: substratusai/base:latest
+  params:
+    epochs: 3
```

Comment:
lets use `num_train_epochs` to so users don't accidentally use `epochs` with our model-trainer-huggingface